.Rbuildignore

```diff
@@ -14,3 +14,5 @@
 ^\.github$
 ^pkgdown$
 ^\.covrignore$
+^\.Rprofile$
```

Comment:
Did you mean to include these?

Reply:
Yep, if you don't mind. It makes my life easier and these should not be in the R package anyway, so it should be harmless for others.